std/encoding.d

```diff
@@ -1528,6 +1526,7 @@ body
     return t.length - s.length;
 }
 
+///
 unittest
 {
     assert(index("\u20AC100",1) == 3);
```

Comment:
I think this unittest needs to be expanded, even if just for documentation purposes.